api/src/main/proto/stellarstation/api/v1/stellarstation.proto

```diff
@@ -316,6 +316,26 @@ message Pass {
   uint64 uplink_center_frequency_hz = 10;
 }
 
+// Data uploaded from the ground station during a pass.
```

Comment:
The term `uploaded` sounds like it puts too much emphasis on the server-client architecture of StellarStation as a software service. Should we use a terminology of satellite communication such as `downlinked`?